plugin/evm/vm_test.go

```diff
@@ -477,3 +480,53 @@ func TestConfigureLogLevel(t *testing.T) {
 		})
 	}
 }
+
+func TestToSet(t *testing.T) {
+	var (
+		idsList []ids.ShortID
+		weights []uint64
+	)
+	for i := 0; i < 3; i++ {
+		weight := rand.Uint64()
+		weights = append(weights, weight)
+
+		id := randID()
+		idsList = append(idsList, id)
+	}
+
+	validators := map[ids.ShortID]uint64{
+		idsList[0]: weights[0],
+		idsList[1]: weights[1],
+		idsList[2]: weights[2],
+	}
+
+	t.Run("Validators", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("nominal case", func(t *testing.T) {
```

Reply:
This is changed also:
https://github.com/flare-foundation/coreth/pull/21/commits/795a19e455a7a1b0b6a427200d807bfb6ec6d3dc#diff-7cc8115837d92067c0e48165d5b49fcd63797639f8680ceee734caa4983e9b32R503